site/docs/spec.md

```diff
@@ -218,10 +218,27 @@ Partition specs capture the transform from table data to partition values. This
 | **`month`**       | Extract a date or timestamp month, as months from 1970-01-01 | `date`, `timestamp(tz)`                                                                                   | `int`       |
 | **`day`**         | Extract a date or timestamp day, as days from 1970-01-01     | `date`, `timestamp(tz)`                                                                                   | `date`      |
 | **`hour`**        | Extract a timestamp hour, as hours from 1970-01-01 00:00:00  | `timestamp(tz)`                                                                                           | `int`       |
+| **`void`**        | Always produces `null` (the void transform)                  | Any                                                                                                       | `null`      |
```

Comment:
`null` isn't really a type. I understand wanting to make it clear what is happening, but I think "Source type" like `identity` is probably more accurate. It is clear that the function result is always null, though.